Update query_program_input.py

vvkrddy opened the pull request:
Renamed headers to RAJ2000, DEJ2000, TYPE where necessary, for consistency across data tables.
Now the Tycho-1 catalogue downloads automatically upto 6 mag without need for user input.
Futher, added user prompt for download of Tycho-1 catalogue in increments of 3 upto 12 mag without need for user enter numbers.
Added warning of large size for Tycho-2 and the download is done only if user agrees.
Apart from above, functionality of the program remains the same.

Commit:
Update query_program_input.py

Renamed headers to RAJ2000, DEJ2000, TYPE where necessary, for consistency across data tables.
Now the Tycho-1 catalogue downloads automatically upto 6 mag without need for user input.
Futher, added user prompt for download of Tycho-1 catalogue in increments of 3 upto 12 mag without need for user enter numbers.
Added warning of large size for Tycho-2 and the download is done only if user agrees.
Apart from above, functionality of the program remains the same.

vvkrddy commented:
Renamed headers to RAJ2000, DEJ2000, TYPE where necessary, for consistency across data tables.
Now the Tycho-1 catalogue downloads automatically upto 6 mag without need for user input.
Futher, added user prompt for download of Tycho-1 catalogue in increments of 3 upto 12 mag without need for user to enter numbers.
Added warning of large size for Tycho-2 and the download is done only if user agrees.
Apart from above, functionality of the program remains the same

Commit:
Update query_program_input.py

Previously, Tycho-1 catalogue up to 9 mag did not append to the file of up to 6 mag.

SahyadriDK commented on Program/query_program_input.py:
```diff
@@ -174,86 +164,85 @@
+    
     while True:
-        if maxV <= 12:
+        if maxV <=12:
```
Why upto 12? If this is downloading all the way upto 12, then there will be redundant entries in the tycho-2 download.
Are we taking it up to 12 only so that there is an option not to download the big file at all?

SahyadriDK reviewed (commented):
Check my comment about maxV<=12.

Commit:
Update query_program_input.py

Now I have made a change so the user can download upto 10.5 mag in tycho-1 and has to download  tycho-2 for upto 12 from 10.5 mag, same as before.
So this should solve the issue of redundant objects and it gives the user only one choice(Tycho-2) for objects within 10.5 and 12 mag.

Also added a break statement in an else statement which sent the program in an infinite loop when the user agrees to download up to the maximum possible magnitude(now 10.5, previously 12).

SahyadriDK reviewed (approved):
Seems good to me. Thanks
